### ISSUE TITLE
Document ALT texts

### ISSUE THREAD

**Issue by TedGoas** [StackExchange/Stacks#349]
Someone noticed I've been putting periods (`.`) at the end of `alt` texts in email, a trick I picked up from [this article](https://axesslab.com/alt-texts/). But I never documented it. 🤦‍♂️

How many other things do we do around `alt` text that need documenting? Shall we start a list?

- [ ] **Always** include and `alt` attribute. If the image is decorative and should be skipped by screenreaders, the `alt` text should be empty (`alt=""`). If an `alt` attribute is missing, screenreaders will read the filename instead, which is rarely helpful.
- [ ] End the alt-text with a period. This will make screen readers pause a bit after the last word in the alt-text, which creates a more pleasant reading experience for the user.
- [ ] Don’t start alt-texts with “Image of”, “Photo of” or similar. The screen reader will add that by default.
- [ ] Keep it concise. There's usually surrounding HTML text that helps describe the image.

**Comment by bnickel** [comment]
We probably want a section around the use of accessibility text with embedded SVG icons.

Eg.

```
<!-- If it's next to relevant text, doing nothing is the same as alt="" -->
<button class="s-btn">
    @Svg.Delete Delete
</button>

<!-- It it's by itself, it should be contained in an element with an aria-label -->
<button class="s-btn" aria-label="Delete">
    @Svg.Delete
</button>
```

**Comment by TedGoas** [comment]
[Done](https://github.com/StackExchange/Stacks/pull/398)
